controller/kubernetes_configmap_controller.go

```diff
@@ -211,6 +211,10 @@ func (kc *KubernetesConfigMapController) reconcile(namespace, cfmName string) er
 		if err := kc.ds.UpdateCustomizedSettings(nil); err != nil {
 			return errors.Wrap(err, "failed to update built-in settings with customized values")
 		}
+	case types.DefaultDefaultBackupStoreSettingsConfigMapName:
+		if err := kc.ds.UpdateDefaultBackupStoreSettings(); err != nil {
+			return errors.Wrap(err, "failed to update default backup store settings with customized values")
+		}
```

Comment:
Do we need to handle the configmap in each reconciliation? Or, we only need to read and create (or update) the default configmap while starting the deamon?

Reply:
>Why do we need to handle the default backupstore configmap in each reconciliation?

The controller will only start to reconcile the ConfigMap `longhorn-default-backupstore` when it is modified.

>Isn't it a one-time job?

No. We need to monitor the ConfigMap when it is modified for these three parameters to update the default backup target as we did in default settings before.
Or we will not allow users to update the default backup target parameters at runtime.

Reply:
I don't think we need to handle this for the default backupstore
1. Fresh installation
    - User sets the parameters for the default backupstore in chart/values.yaml
    - app/daemon.go creates the default backupstore based on the parameters
    - If user updates the configmap, we don't need to sync them
      - Any benefit that we sync the configmap to the default backupstore?
 2. Upgrade
    - Case 1
      -  User updates the parameters for the default backupstore in chart/values.yaml
      - app/daemon.go updates the default backupstore based on the parameters
    - Case 2
      - User doesn't set the parameters for the default backupstore in chart/values.yaml
      - app/daemon.go doesn't need to update the default backupstore